core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -180,7 +194,7 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
   }
 
   def schedulePollTask(): Cancellable =
-    context.system.scheduler.scheduleOnce(pollInterval(), self, Poll)(context.dispatcher)
+    context.system.scheduler.scheduleOnce(pollInterval(), self, Poll(self))(context.dispatcher)
```

Comment:
to avoid creating a new `Poll` instance all the time:
`val pollMsg = Poll(this)`